ninja-standalone/src/main/java/ninja/standalone/NinjaJetty.java

```diff
@@ -90,111 +118,196 @@ public Injector getInjector() {
+            
+            String[] configs = this.jettyConfiguration.split(",");
+            
+            for (String config : configs) {
+            
+                server = createServerOrApplyConfiguration(config, server);
+                
+            }
+            
+        }
+        else {
+            
+            // create very simple jetty configuration
+             server = new Server();
+        
+            // HTTP connector
+            ServerConnector http = new ServerConnector(server);
 
-        try {
+            http.setPort(port);
+
+            if (host != null) {
+                http.setHost(host);
+            }
+
+            // set the connector
+            server.addConnector(http);
             
-            context = new ServletContextHandler(server, ninjaContextPath);
+        }
             
-            NinjaPropertiesImpl ninjaProperties 
-                    = new NinjaPropertiesImpl(ninjaMode);
-            // We are using an embeded jetty for quick server testing. The
-            // problem is that the port will change.
-            // Therefore we inject the server name here:
-            ninjaProperties.setProperty(NinjaConstant.serverName, serverUri.toString());
+        context = new ServletContextHandler(server, ninjaContextPath);
+        
+        ninjaProperties = new NinjaPropertiesImpl(ninjaMode);
 
-            ninjaServletListener.setNinjaProperties(ninjaProperties);
+        // We are using an embeded jetty for quick server testing. The
+        // problem is that the port will change.
+        // Therefore we inject the server name here:
+        String serverName = (this.serverUri != null
+                                ? this.serverUri.toString() : createServerName());
+        
+        ninjaProperties.setProperty(NinjaConstant.serverName, serverName);
 
-            context.addEventListener(ninjaServletListener);
+        ninjaServletListener.setNinjaProperties(ninjaProperties);
+        ninjaServletListener.setLogInjectorException(false);
 
-            context.addFilter(GuiceFilter.class, "/*", null);
-            context.addServlet(DefaultServlet.class, "/");
+        context.addEventListener(ninjaServletListener);
+        context.addFilter(GuiceFilter.class, "/*", null);
+        context.addServlet(DefaultServlet.class, "/");
+            
+        try {
 
             server.start();
 
-        } catch (Exception ex) {
-            throw new RuntimeException(ex);
+        } catch (Exception cause) {
+            
+            // check if there was an underlying injector exception (most likely cause)
+            if (ninjaServletListener.getNinjaBootstrap() != null &&
+                    ninjaServletListener.getNinjaBootstrap().getInjectorException() != null) {
+                
+                // the injector exception is actually what we want thrown
+                throw ninjaServletListener.getNinjaBootstrap().getInjectorException();
+                
+            }
```

Comment:
Ninja's codebases uses else on the same line as "}" like

```
} else {
```

That may sound really picky... but uniformity is important in the long run as the codebase grows :) Thanks!